app/src/components/Button.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import styles from "./styles.module.scss";
+
+function Button({ text, handleFxn }) {
```

Comment:
Suggest to namespace this to your project so it doesn't cause confusion to outside observers

`Button` is generic